src/refiners/EN/ENPrioritizeSpecificDateRefiner.js

```diff
@@ -0,0 +1,109 @@
+/*
+
+*/
+var ParsedComponents = require('../../result').ParsedComponents;
+var Refiner = require('../refiner').Refiner;
+
+
+var PATTERN = new RegExp("^\\s*(at|after|before|on|,|-|\\(|\\))?\\s*$");
+
+function isMoreSpecific(prevResult, currResult) {
+    var moreSpecific = false;
+
+    if (prevResult.start.isCertain('year'))
```

Comment:
Could you please add `{ }` on all `if` and `else` clue, even those one-line ones.